src/utils/helpers.ts

```diff
@@ -10,3 +12,25 @@ export async function sleep(timeMs: number): Promise<null> {
     setTimeout(resolve, timeMs);
   });
 }
+
+export function currentTimeInSeconds(): number {
```

Comment:
we have https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/transactions/management/accountSequenceNumber.ts#L32 - maybe can reuse?